Apromore-Custom-Plugins/User-Admin-Portal-Plugin/src/main/java/org/apromore/plugin/portal/useradmin/UserAdminController.java

```diff
@@ -265,7 +265,7 @@ public void doAfterCompose(Window win) throws Exception {
         mainWindow = win;
         String userName = portalContext.getCurrentUser().getUsername();
         currentUser = securityService.getUserByName(userName);
-        selectedUser = currentUser;
+        selectedUser = null;
```

Comment:
While this change isn't harmful, it seems a little less convenient that the old default of showing someone their own account when they open the user management window.